.travis.yml

```diff
@@ -10,10 +10,14 @@ python:
   - "2.7"
 
 # command to install dependencies
-install: "pip install -Ue .[test,build]"
+install: 
+  - "pip install -Ue .[test,build]"
+  - pip install pep8==1.5.7
```

Comment:
Huh? pep8 not a testing dependency then? ;-)
Add it in the right place if you want to use it...